kolibri/core/assets/src/core-store.js

```diff
@@ -135,7 +135,15 @@ const mutations = {
     }
     state.core.logging.mastery.end_timestamp = currentTime;
   },
-  UPDATE_LOGGING_ATTEMPT(state, currentTime, correct, complete, hinted) {
+  UPDATE_LOGGING_ATTEMPT(state, {
```

Comment:
I think there is a lot of potential to rework this, but I was making the minimal change here.